pipelines/shared/Jenkinsfile

```diff
@@ -82,33 +82,29 @@ pipeline {
                     echo 'Do development deployment'
                     commonModule = load "pipelines/shared/common.groovy"
                     intention = new BrokerIntention(readJSON(file: "app/intention.json"))
-                    intention.userName = env.CAUSE_USER_ID
-                    intention.eventUrl = env.BUILD_URL
-                    intention.eventProvider = env.EVENT_PROVIDER
-                    intention.intention.actions[1].service.environment = 'development'
-                    intention.intention.actions[2].service.environment = 'development'
-                    intention.intention.actions[2].package.version = env.PODMAN_CD_VERSION
+                    intention.setEventDetails(
+                        userName: env.CAUSE_USER_ID,
+                        url: env.BUILD_URL,
+                        provider: env.EVENT_PROVIDER,
+                        environment: 'development',
+                        packageInstallationVersion: env.PODMAN_CD_VERSION
+                    )
                     intention.open(NR_BROKER_JWT)                    
                     intention.startAction("login")
                     def vaultToken = intention.provisionToken("login", CONFIG_ROLE_ID)
                     def vault = new Vault(vaultToken)
-                    def cdCreds = vault.read('groups/data/appdelivery/jenkins-isss-cdua')
-                    env.PODMAN_CD_USER = cdCreds['username_lowercase']
-                    env.PODMAN_CD_PASS = cdCreds['password']
-                    def registryCreds = vault.read("${env.ARTIFACTORY_VAULT_PATH}")
-                    env.REGISTRY_USERNAME = registryCreds['username']
-                    env.REGISTRY_PASSWORD = registryCreds['password']
+                    vault.readToObject('groups/data/appdelivery/jenkins-isss-cdua', cdCreds, keyTransform: { key -> key.toUpperCase() })
+                    env.PODMAN_CD_USER = cdCreds.USERNAME_LOWERCASE
+                    env.PODMAN_CD_PASS = cdCreds.PASSWORD
+                    vault.readToObject("${env.ARTIFACTORY_VAULT_PATH}", registryCreds, keyTransform: { key -> key.toUpperCase() })
```

Comment:
vault.readToObject("${env.ARTIFACTORY_VAULT_PATH}", env, keyTransform: { key -> 'REGISTRY_' + key.toUpperCase() })
and remove 100 & 101